packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/tests/useBlocksStore.test.js

```diff
@@ -782,6 +782,74 @@ describe('useBlocksStore', () => {
     ]);
   });
 
+  it('handles the backspace key on a list with two list items and converts the first into a paragraph', () => {
+    const { result } = renderHook(useBlocksStore);
+
+    baseEditor.children = [
+      {
+        type: 'list',
+        format: 'ordered',
+        children: [
+          {
+            type: 'list-item',
+            children: [
+              {
+                type: 'text',
+                text: 'first list item',
+              },
+            ],
+          },
+          {
+            type: 'list-item',
+            children: [
+              {
+                type: 'text',
+                text: 'second list item',
+              },
+            ],
+          },
+        ],
+      },
+    ];
+
+    // Set the cursor on the first list item
+    Transforms.select(baseEditor, {
+      anchor: { path: [0, 0, 0], offset: 0 },
+      focus: { path: [0, 0, 0], offset: 0 },
+    });
+
+    // Simulate the backspace key
+    result.current['list-ordered'].handleBackspaceKey(baseEditor, mockEvent);
+
+    // Should convert the first list item in a paragraph
+    expect(baseEditor.children).toEqual([
+      {
+        type: 'paragraph',
+        children: [
+          {
+            type: 'text',
+            text: 'first list item',
+          },
+        ],
+      },
+      {
+        type: 'list',
+        format: 'ordered',
+        children: [
+          {
+            type: 'list-item',
+            children: [
+              {
+                type: 'text',
+                text: 'second list item',
+              },
+            ],
+          },
+        ],
+      },
+    ]);
+  });
```

Comment:
Are we sure that's what we want? I would assume that the first item would be completely deleted and we'd be left with just a list with the second list item. I thought the case where we wanted to convert the list to a paragraph is if there is no list items anymore

Reply:
I disagree with that, Notion, JIra don't work like you said. If you have a list and you click backspace on the first list item it will become a paragraph. Maybe we can involve @yanniskadiri in the discussion

Reply:
that seemed weird to me on paper, but looks quite natural in your video. I'm good to go ahead with it 👍

Reply:
That works like this for me too